tests/ui/generic-associated-types/assume-gat-normalization-for-nested-goals.next.stderr

```diff
@@ -2,7 +2,7 @@ error[E0277]: the trait bound `i32: Baz<Self>` is not satisfied
   --> $DIR/assume-gat-normalization-for-nested-goals.rs:9:30
    |
 LL |     type Bar<T>: Baz<Self> = i32;
-   |                              ^^^ the trait `Eq<i32>` is not implemented for `i32`, which is required by `i32: Baz<Self>`
```

Comment:
This message was also previously just not correct -- `Eq<i32>` is implemented by `i32` lol